docs/getting_started/tutorial.md

```diff
@@ -652,16 +651,16 @@ Here, you use the `{% image %}` tag, which exists in the `wagtailimages_tags` li
 
 !["Second Post" page, with title, date, intro, body, and a gallery of three images](../_static/images/tutorial/tutorial_6.png)
 
-Since your gallery images are database objects in their own right, you can now query and re-use them independently of the blog post body. Now, define a `main_image` method, which returns the image from the first gallery item or `None` if no gallery items exist:
+Since your gallery images are database objects in their own right, you can 
+now query and re-use them independently of the blog post body. Now, define a `main_image` method in your `BlogPage` model, which returns the image from the first gallery item or `None` if no gallery items exist:
```

Comment:
Unless there’s a reason for it let’s avoid splitting line manually, so people can set their word-wrapping when editing text to what they prefer.

```suggestion
Since your gallery images are database objects in their own right, you can  now query and re-use them independently of the blog post body. Now, define a `main_image` method in your `BlogPage` model, which returns the image from the first gallery item or `None` if no gallery items exist:
```